### DISCUSSION TITLE
Expanding DB size

### DISCUSSION THREAD

**Discussion by Drewq2004** [mealie-recipes/mealie#1885] [General]
For those of us that use a NAS is there a way to expand the Database size to larger the 500MB or is the size variable and thus grows as you add more?

**Comment by michael-genson** [comment]
That feature isn't implemented yet, so you can safely go past the 500MB limit. Currently it's just a reference

**Comment by Drewq2004** [comment]
Michael,


Thank you for the help.

________________________________
